marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/entity-lib.sjs

```diff
@@ -263,6 +263,15 @@ function findModelReferencesInOtherModels(entityModelUri, entityTypeId) {
   return [...affectedModels];
 }
 
+function findForeignKeyReferencesInOtherModels(entityModel){
+  const entityTypeId = getEntityTypeId(entityModel, entityModel.info.title);
+  const entityModelsWithForeignKeyReferences = cts.search(cts.andQuery([cts.collectionQuery(consts.ENTITY_MODEL_COLLECTION),
```

Comment:
@ryanjdew Should a sparql query here be possible, and presumably more efficient too since we wouldn't be loading docs? 

@srinathgit  Check what's in /v1/graphs/things when you run your test - I think ML may project triples from the entity model automatically that allow for using sparql here.

Reply:
@rjrudin 
I looked into the triples in "v1/graphs?graph=http://marklogic.com/entity-services/models". There aren't any triples for properties that we define like "relatedEntityType", "facetable", "sortable".